src/ops/mod.rs

```diff
@@ -55,7 +55,7 @@ impl ExitError {
     /// Helpers to create exit results
     ///
     /// Note: err panics if exitcode is zero.
-    fn err<T>(exitcode: i32, message: T) -> ExitError
+    pub fn err<T>(exitcode: i32, message: T) -> ExitError
```

Comment:
Let's make a function which creates an error of exitcode 100, with a variable message, named to represent what that error code represents.

Reply:
Not actually using this in this commit, leftover from splitting out of the other PR probably. Will remember it when the merge conflicts hit.